### DISCUSSION TITLE
Robotic sound, Glow TTS+vocoder

### DISCUSSION THREAD

**Discussion by Huba2004** [coqui-ai/TTS#2125] [General Q&A]
I' m trying to teach TTS in Hungarian.
I have about a 1,5 h recording ready. I used Glow TTS and vococer for the training. The first file is without vocoder, the second is Glow TTS + vocoder. What did I do wrong?

(tts text -- "Magyar nyelvű teszt mondat.")

Output1: https://soundcloud.com/huba-szalkai/output/s-QlGUFScsuqV?si=c7ea0702832c42ff933bde77d8a0185c&utm_source=clipboard&utm_medium=text&utm_campaign=social_sharing&fbclid=IwAR3mkT86l66OLtW1x9Suco1ZWu6cJsRAgC7kkyEaFHhsYiLAAjB5puNG5Wk

Output2: https://soundcloud.com/huba-szalkai/output2-1/s-ObMLaTqcJaB?si=c7ea0702832c42ff933bde77d8a0185c&utm_source=clipboard&utm_medium=text&utm_campaign=social_sharing&fbclid=IwAR2pef6skRoxHleEmj3tK23mykk0exxG87Ngu7rXtwPBbOjazIct5qcVqR8

**Comment by erogol** [comment]
It is impossible to comment on that by just checking the output files.

**Reply by Huba2004** [reply]
Thank you for your reply! What additional files do I need to upload?
